packages/inter-protocol/test/vaultFactory/test-vaultLiquidation.js

```diff
@@ -2398,7 +2398,7 @@ test('Bug 7346 excess collateral to holder', async t => {
   t.false(await E(bidderSeat3).hasExited());
   await E(bidderSeat3).tryExit();
   t.true(await E(bidderSeat3).hasExited());
-  await assertBidderPayout(t, bidderSeat3, run, 10_460n, aeth, 0n);
+  await assertBidderPayout(t, bidderSeat3, run, 10_460n, aeth, 16432n);
```

Comment:
I'm having a hard time validating that this value is correct. Can you name some of these values and express them in terms of each other?

Reply:
Yes. Take another look. I multiplied everything out to match the numbers in @rowgraus' scenario, and added a lot of details.